### PR TITLE
add models to NLP example

### PR DESCRIPTION
**Pull request by violetch24**
## Type of Change
add models for huggingface model zoo to NLP examples
refine tc-qat README
remove deberta model for now
## Description
JIRA: https://jira.devtools.intel.com/browse/ILITV-2733
Please refer to: https://intel-my.sharepoint.com/:x:/p/zixuan_cheng/ETYoWw3HUtJAhMUtEOcp0-4BkkWTtOmJtk8Vw7E_OfhPVQ

## Expected Behavior & Potential Risk
N/A

## How has this PR been tested?
extension test

**topologies in this PR**: distilbert_base_MRPC_dynamic, albert_base_MRPC, bert_base_MRPC, bert_base_CoLA, bert_base_STS-B, bert_base_SST-2, bert_base_RTE, bert_large_QNLI, bert_large_CoLA, distilbert_base_MRPC, xlnet_base_cased_MRPC, roberta_base_MRPC, camembert_base_MRPC, bert_base_MRPC, gpt_neox

## Dependency Change?
N/A

### CODE REVIEW

File: examples/.config/model_params_pytorch.json [hunk 1]
```diff
@@ -415,29 +409,25 @@
        "batch_size": 1
     },
     "distilbert_base_MRPC": {
```
Comment (chensuyue) [inline examples/.config/model_params_pytorch.json:411]: update model name into `distilbert_base_MRPC_dynamic`

File: examples/.config/model_params_pytorch.json [hunk 2]
```diff
@@ -415,29 +409,25 @@
...
       "dataset_location": "/tf_dataset/pytorch/glue_data_new",
-      "input_model": "/tf_dataset/pytorch/huggingface/language_translation_pt/distilbert_mrpc",
-      "yaml": "conf.yaml",
-      "strategy": "basic",
+      "input_model": "/tf_dataset2/models/huggingface/distilbert-base-uncased-MRPC",
       "batch_size": 64,
-      "new_benchmark": false
+      "main_script": "run_glue.py"
     },
     "distilbert_base_MRPC_fx": {
```
Comment (chensuyue) [inline examples/.config/model_params_pytorch.json:418]: update model name into `distilbert_base_MRPC`

File: examples/pytorch/nlp/huggingface_models/text-classification/quantization/ptq_static/fx/run_tuning.sh [hunk 3]
```diff
@@ -45,6 +45,30 @@ function run_tuning {
...
+    elif [ "${topology}" = "bert_base_SST-2" ]; then
+        TASK_NAME='sst2'
+    elif [ "${topology}" = "bert_base_RTE" ]; then
+        TASK_NAME='rte'
+    elif [ "${topology}" = "bert_large_QNLI" ]; then
+        TASK_NAME='qnli'
+    elif [ "${topology}" = "bert_large_CoLA" ]; then
+        TASK_NAME='cola'
+    elif [ "${topology}" = "distilbert_base_MRPC_fx" ]; then
+        TASK_NAME='mrpc'
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/text-classification/quantization/ptq_static/fx/run_tuning.sh:63]: elif [ "${topology}" = "distilbert_base_MRPC"* ]; then

**Comment by violetch24** [comment]
**topologies to be tested**: distilbert_base_MRPC_dynamic, distilbert_base_MRPC, gpt_neox

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4627/artifact/report.html)

**Comment by violetch24** [comment]
**topologies to be tested**: distilbert_base_MRPC_dynamic, distilbert_base_MRPC

**Comment by chensuyue** [comment]
> **topologies to be tested**: distilbert_base_MRPC_dynamic, distilbert_base_MRPC

[retest](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4645/)

**Comment by chensuyue** [comment]
Test pass, please update all models into readme https://github.com/intel/neural-compressor/blob/zixuan/nlp_models/examples/README.md, and we can merge this PR.

**Pull request merged**